scripts/phase8/llm_background_load.py

```
@@ -0,0 +1,133 @@
+    explicit = os.getenv("B07_P8_RUNTIME_DATABASE_URL")
+    if explicit:
+        return explicit
+    runtime = os.getenv("DATABASE_URL")
+    if not runtime:
+        raise RuntimeError("DATABASE_URL or B07_P8_RUNTIME_DATABASE_URL is required")
+    if runtime.startswith("postgresql+asyncpg://"):
+        return runtime.replace("postgresql+asyncpg://", "postgresql://", 1)
+    return runtime
+
+
+def _seed_tenant(runtime_db_url: str) -> UUID:
```

Ruff (0.15.0)

[error] Undefined name `UUID`

(F821)